geoh5py/data/__init__.py

```diff
@@ -45,3 +46,21 @@
 from .text_data import CommentsData, MultiTextData, TextData
 from .unknown_data import UnknownData
 from .visual_parameters import VisualParameters
+
+
+class TypeEnum(Enum):
```

Comment:
it's seems a duplicate with PrimitiveTypeEnum no?
What the difference?
seems redundant